rootfs/Dockerfile

```diff
@@ -42,6 +42,9 @@ RUN bash -eu -c ' \
 RUN  setcap    cap_net_bind_service=+ep /nginx-ingress-controller \
   && setcap -v cap_net_bind_service=+ep /nginx-ingress-controller
 
+RUN  setcap    cap_net_bind_service=+ep /usr/sbin/nginx \
+  && setcap -v cap_net_bind_service=+ep /usr/sbin/nginx
```

Comment:
What's this for?

Reply:
talked in Slack: this is to fix https://github.com/kubernetes/ingress-nginx/issues/3732